client/signup/main.jsx

```diff
@@ -318,7 +318,12 @@ class Signup extends React.Component {
 		}
 	}
 
-	handleSignupFlowControllerCompletion = ( dependencies, destination ) => {
+	handleSignupFlowControllerCompletion = async ( dependencies, destination ) => {
+		// See comment below for `this.bizxSurveyTimerComplete`
+		if ( this.bizxSurveyTimerComplete ) {
```

Comment:
Would it be possible to also use an if like the [one here](https://github.com/Automattic/wp-calypso/blob/master/client/lib/analytics/hotjar.js#L19) so that we don't slow down the test for users who are not going to see the hotjar?

Reply:
That's a good suggestion! I added a check instead to check if the hotjar script has been loaded.